src/content/docs/blog/caching-update.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: The Caching Update
+description: We've been working hard to make your building experience smoother!
+date: 2024-04-2X
```

Comment:
```suggestion
date: 2024-04-27
```

The builds are failing due to this. The date should be valid, without the `X` placeholder. It can be set to a dummy date for now, then updated for the actual publish date right before merge.